http/h2_error.lua

```diff
@@ -52,7 +52,7 @@ function http_error_methods:new_traceback(message, stream_error, lvl)
 	if lvl ~= 0 then
 		-- COMPAT: should be passing `nil` message (not the empty string)
 		-- see https://github.com/keplerproject/lua-compat-5.3/issues/16
-		e.traceback = debug.traceback("", lvl)
+		e.traceback = debug and debug.traceback("", lvl) or "no debug.traceback"
```

Comment:
I don't consider this valid: a function `:new_traceback` should *always* gather a traceback or throw an error